### PR TITLE
feature/MAUI-APP#76

### PR DESCRIPTION
**Pull request by jacques-andre**

### CODE REVIEW

**Comment by calvin1337** [comment]
Fix the conflicts. merge your branch with the latest version of develop before submitting a pull request.

Put a message on discord or dm me on discord if you need a review I have only just saw that you requested a review and apologise.

File: Services/ITSystemStatusService.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+using MauiApp1.Models;
+using SQLite;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MauiApp1.Services
```
Comment (calvin1337) [inline Services/ITSystemStatusService.cs:9]: The application has been renamed. change the name space too UndacApp.Services

Comment (jacques-andre) [reply Services/ITSystemStatusService.cs]: Thanks. Updated.

**Review by calvin1337 — APPROVED** [review]
Everything looks good to me!

**Pull request merged**